templates/components/vectordbs/python/llamacloud/service.py

```diff
@@ -79,12 +80,14 @@ def add_file_to_pipeline(
         max_attempts = 20
         attempt = 0
         while attempt < max_attempts:
-            result = client.pipelines.get_pipeline_file_status(pipeline_id, file.id)
+            result = client.pipelines.get_pipeline_file_status(
+                file_id=file_id, pipeline_id=pipeline_id
```

Comment:
The get_pipeline_file_status() function has changed the ordering of the parameters. It is better to use keyword arguments.

Reply:
we should add a parameter to not wait for the file being processed. then from `generate.py` we don't wait and when the user is uploading a file, we wait

Reply:
Thanks! I just remembered this.